src/DynamicRupture/Output/ReceiverBasedOutput.cpp

```diff
@@ -88,6 +88,7 @@ void ReceiverOutput::calcFaultOutput(
 #pragma omp parallel for
 #endif
   for (size_t i = 0; i < outputData->receiverPoints.size(); ++i) {
+    // TODO: query the dofs, only once per simulation
```

Comment:
That's not only a TODO for once per simulation, but once per face.
(note: the GPU aggregators already do exactly that; but on the CPU, we copy for each point again)